tests/mock_tables/database_config.json

```diff
@@ -0,0 +1,57 @@
+{
+    "INSTANCES": {
+        "redis":{
+            "hostname" : "127.0.0.1",
+            "port" : 6379,
+            "unix_socket_path" : "global_db"
```

Comment:
>global_db [](start = 34, length = 9)

seems a hack

Reply:
Fixed as discussed, mocked the init function of SonicV2Connector class to add a new field in kwargs called "namespace". This will help form the right json file name to be loaded in mockRedis.